packages/components/src/Link/Link.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { cleanup, render } from "@testing-library/react";
+import { Link } from ".";
+
+afterEach(cleanup);
+
```

Comment:
We would typically move the content being `render`ed and `getBy`ed into a variable, for readability and to be more robust (as it uses the same variable in the comparison). E.g. 
```
const testURL = "https://getjobber.com";
const testLinkText = "This is a link";

describe("when a Link is rendered", () => {
  it("opens the link in the same tab if external is left at default value of false", () => {
    const { getByText } = render(<Link url={testURL}>{testLinkText}</Link>);
    const link = getByText(testLinkText);
    expect(link.getAttribute("target")).toBeNull();
  });
...
// other tests
});
```